.github/workflows/device-discovery-lint-tests.yaml

```diff
@@ -46,7 +50,7 @@ jobs:
         pytest --junitxml=pytest.xml --cov-report=term-missing:skip-covered --cov=device_discovery/ | tee pytest-coverage.txt
 
     - name: Pytest coverage comment
-      uses: MishaKav/pytest-coverage-comment@main
+      uses: MishaKav/pytest-coverage-comment@81882822c5b22af01f91bd3eacb1cefb6ad73dc2
```

Comment:
Could we add (across all affected workflow in the PR) a trailing comment with a version the commit hash is for? See examples in diode/diode-netbox-plugin, etc.

Reply:
sure

Reply:
done